server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -185,14 +184,16 @@ public enum EventType implements EnumWithStableIndex {
     REMOVE_CREDENTIAL_ERROR(0x10000 + REMOVE_CREDENTIAL.getStableIndex(), true),
 
     INVITE_ORG(60, true),
-    INVITE_ORG_ERROR(0x10000 + INVITE_ORG.getStableIndex(), true);
+    INVITE_ORG_ERROR(0x10000 + INVITE_ORG.getStableIndex(), true),
+
+    USER_SESSION_DELETED(61, false)
```

Comment:
Do we need a corresponding `USER_SESSION_DELETED_ERROR` enum, it seems all other event types have this?

Reply:
lets add it for symmetry reasons.